src/faostat_data_primap/download.py

```diff
@@ -31,7 +173,9 @@ def get_html_content(url: str) -> bs4.BeautifulSoup:
     # will automatically download it for you. Make sure there is no
```

Comment:
Why make sure there is no chrome driver installed? Typo?

Reply:
Sometimes Selenium asks you to update your Chrome driver to the latest version. With the latest version of Selenium you don't need to do this anymore and Selenium will download the correct driver for you. However, that only works if you do not have a Chrome driver version installed on your system. I got an error that said I have to delete my Chrome driver. But, I can see how this comment may be more confusing than helpful, so I think it's better to delete it.